### PR TITLE
Updated credits

### PR DESCRIPTION
**Pull request by AleksMat**
Updated credits with new members and new time intervals.

### CODE REVIEW

File: features/eolearn/features/clustering.py [hunk 1]
```diff
@@ -1,5 +1,12 @@
 """
 Module for computing clusters in EOPatch
+
+Credits:
+Copyright (c) 2020 Beno Šircelj (Josef Stefan Institute)
```
Comment (zigaLuksic) [inline features/eolearn/features/clustering.py:5]: usually you have them sorted by end dates

File: features/eolearn/features/doubly_logistic_approximation.py [hunk 2]
```diff
@@ -1,5 +1,12 @@
 """
 Module for calculating doubly logistic approximation.
+
+Credits:
+Copyright (c) 2020 Beno Šircelj (Josef Stefan Institute)
```
Comment (zigaLuksic) [inline features/eolearn/features/doubly_logistic_approximation.py:5]: again order seems different

File: features/eolearn/tests/test_clustering.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
+"""
+Credits:
+Copyright (c) 2020 Beno Šircelj (Josef Stefan Institute)
```
Comment (zigaLuksic) [inline features/eolearn/tests/test_clustering.py:3]: here as well

Comment (zigaLuksic) [reply features/eolearn/tests/test_clustering.py:3]: just check everywhere for this kind of docstring, seems to be this one everywhere

File: features/eolearn/tests/test_haralick.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 """
 Credits:
 Copyright (c) 2018-2019 Hugo Fournier (Magellium)
```
Comment (zigaLuksic) [inline features/eolearn/tests/test_haralick.py:3]: order wrong here as well, search for this one as well as it repeats

File: features/eolearn/tests/test_radiometric_normalization.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 """
 Credits:
 Copyright (c) 2018-2019 Johannes Schmid (GeoVille)
```
Comment (zigaLuksic) [inline features/eolearn/tests/test_radiometric_normalization.py:3]: wrong order

File: io/eolearn/io/extra/geodb.py [hunk 6]
```diff
@@ -4,7 +4,7 @@
 To use tasks from this module you have to install dependencies defined in `requirements-geodb.txt`.
 
 Credits:
-Copyright (c) 2017-2021 Matej Aleksandrov, Matej Batič (Sinergise)
+Copyright (c) 2021-2022 Matej Aleksandrov, Matej Batič (Sinergise)
```
Comment (zigaLuksic) [inline io/eolearn/io/extra/geodb.py:7]: shouldn't this be 2017-2022?

File: features/eolearn/features/clustering.py (hunk 1 above)
Comment (AleksMat) [reply features/eolearn/features/clustering.py:5]: I wouldn't change this for now as the first one was the main contributor.

File: io/eolearn/io/extra/geodb.py (hunk 6 above)
Comment (AleksMat) [reply io/eolearn/io/extra/geodb.py:7]: We actually only added this module last year so I wrote 2021-2022

**Pull request merged**